53-aditofsdk.rules

```diff
@@ -0,0 +1,12 @@
+# allow read/write access to the eeprom
+
+SUBSYSTEM=="i2c", 
+RUN+="/bin/chown root:i2c /sys/bus/i2c/devices/0-0056/eeprom", 
+RUN+="/bin/chmod 0666 /sys/bus/i2c/devices/0-0056/eeprom", 
+RUN+="/bin/chown root:i2c /sys/bus/i2c/devices/1-0056/eeprom", 
+RUN+="/bin/chmod 0666 /sys/bus/i2c/devices/1-0056/eeprom" ,
+RUN+="/bin/chown root:i2c /sys/bus/i2c/devices/6-0056/eeprom", 
+RUN+="/bin/chmod 0666 /sys/bus/i2c/devices/6-0056/eeprom" ,
+RUN+="/bin/chmod 666 /dev/i2c-1" ,
+RUN+="/bin/chmod 666 /dev/i2c-0",
+RUN+="/bin/chmod 666 /dev/i2c-6" ,
```

Comment:
I thought that you'll do a udev_rules file for each board that we support. Is this a combination for all of them?

Reply:
yes, this is a combination. Should I separate them?

Reply:
yes

Reply:
solved. thanks